controller/token.go

```diff
@@ -142,25 +143,60 @@ func (c *TokenController) Generate(ctx *app.GenerateTokenContext) error {
 		}, "unable to get Generate User token")
 		return jsonapi.JSONErrorResponse(ctx, errors.NewInternalError(ctx, errs.Wrap(err, "unable to generate test token ")))
 	}
-	_, _, err = c.Auth.CreateOrUpdateIdentity(ctx, *testuser.Token.AccessToken)
+
+	identity, _, err := c.Auth.CreateOrUpdateIdentity(ctx, *testuser.Token.AccessToken)
+	if err != nil {
+		log.Error(ctx, map[string]interface{}{
+			"err": err,
+		}, "unable to persist user properly")
+		return jsonapi.JSONErrorResponse(ctx, errors.NewInternalError(ctx, errs.Wrap(err, "unable to persist user properly")))
```

Comment:
This line is the reason of test failure. There is no public key available in the token manager used by tests to parse token properly. So the CreateOrUpdate function fails in tests.

Reply:
It's quite tricky to fix tests if we want to fails here. It requires some big refactoring in our tests. So, we could just ignore this error for now. Merge my PR to your topic branch for that - https://github.com/kishansagathiya/fabric8-auth/pull/1
I'm not authorized to push to your branch.

Reply:
@alexeykazakov Merged https://github.com/kishansagathiya/fabric8-auth/pull/1. Thanks a lot for this.